### DISCUSSION TITLE
How can I prevent non-collaborators from using the "Assign" feature of Issues?

### DISCUSSION THREAD

**Discussion by MartinThoma** [community/community#81697] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

People tend to assign me in issues of pypdf. That is confusing and not helpful. I don't want that. It's annoying. How can I prevent it?

Either I want to disable that feature completely, or I want non-collaborators to be prevented from using it.

**Comment by ananuness** [comment]
As anyone with write access can assign issues and pull requests, it's strange a non-collaborator have this permission, but you can check the interaction permissions in `Settings > Moderation options > interactions limits` (this is not definitive).
